src/pytest_inmanta_lsm/lsm_project.py

```diff
@@ -938,12 +939,14 @@ def post_partial_compile_validation(
         resource_sets = get_resource_sets(self.project)
         if not service.deleted:
             # Check that the only resource set emitted is the one of this service
-            assert resource_sets.keys() == {str(service_id)}
-            _, owned_resources = resource_sets.popitem()
+            assert str(service_id) in resource_sets.keys()
```

Comment:
This doesn't always hold (for systemtenant, we always only emit the resource set of the systemtenant).

Reply:
How about the lifecycle transfers of the different services, are they also in the st resource set?